test/integration/bootstrap/bootstrap_test.go

```diff
@@ -15,23 +15,171 @@
 package bootstrap
 
 import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
 	"testing"
 	"time"
 
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
 	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
+	"github.com/gruntwork-io/terratest/modules/terraform"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"github.com/terraform-google-modules/enterprise-application/test/integration/testutils"
 )
 
+// fileExists check if a give file exists
+func fileExists(filePath string) (bool, error) {
```

Comment:
Should this function go in? https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/tree/main/test/integration/testutils